ballerina/tests/02-execute-basic-test.bal

```diff
@@ -20,12 +20,13 @@ import ballerina/test;
     groups:["execute", "execute-basic"]
 }
 isolated function testCreateTable() returns sql:Error? {
-    Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT);
-    sql:ExecutionResult result = check dropTableIfExists("TestExecuteTable");
+    sql:ConnectionPool pool = {maxOpenConnections: 3, minIdleConnections: 1};
```

Comment:
shall we move the `connection pool` variable to module level and use it in all clients. So the same connection pool is shared by all clients

Reply:
AFAIU, when we are trying to create a client object using `Client oracledbClient = check new(HOST, USER, PASSWORD, DATABASE, PORT);` it creates an `oracledbClient` object with a globalConnectionPool. since we are adding new test cases to the module, this leads to use of the same Hikari Connection pool across a lot of test cases even we create new Clients. There some test cases can not run due to random connection Timeout happen when a lot of test cases try to get connections from the same Hikari connection pool. that is the reason for adding local connection pool variable when creating Clients. We can prioritize moving table creation and type creation to SQL scripts so to reduce the total connections used.